### PR TITLE
Release 1.17.0-RC2

### PR DESCRIPTION
**Pull request by xprazak2**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [x] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
Not sure why annex is failing here.

[test packaging]

**Comment by ares** [comment]
From what I see in koji
```
webpack --bail --config config/webpack.config.js
'
module.js:471
    throw err;
    ^
Error: Cannot find module 'uglifyjs-webpack-plugin'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/builddir/build/BUILD/foreman-1.17.0-RC2/config/webpack.config.js:6:22)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
RPM build errors:
error: Bad exit status from /var/tmp/rpm-tmp.y0FkzT (%build)
```

**Comment by ekohl** [comment]
Oh yes, I needed to look at https://github.com/theforeman/foreman-packaging/pull/2278

**Comment by ares** [comment]
[test packaging][test]

**Comment by ekohl** [comment]
Something is not right in the build process. `Checking for tag [foreman-installer-1.17.0-0.1.RC1.fm1_17] in git repo [git://github.com/theforeman/foreman-packaging.git]` suggests to me that it's using the newest tito which contains a regression.

**Comment by ekohl** [comment]
[test packaging]

File: foreman/foreman.spec [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 # set and uncomment all three to set alpha tag
-%global alphatag RC1
+%global alphatag RC2
 %global dotalphatag .%{alphatag}
 %global dashalphatag -%{alphatag}
 
 Name:   foreman
 Version: 1.17.0
-Release: 0.3%{?dotalphatag}%{?dist}
+Release: 0.1%{?dotalphatag}%{?dist}
```
Comment (ekohl) [inline foreman/foreman.spec]: You should not decrease this because the package will be seen as older since 1.17.0-0.1RC2 < 1.17.0-0.3RC1.

**Comment by ehelms** [comment]
This should only work with `--test` because these builds lack tito tags (which are traditionally done after merge). Did the job somehow regress to drop `--test` ?

**Comment by ekohl** [comment]
> This should only work with --test because these builds lack tito tags (which are traditionally done after merge). Did the job somehow regress to drop --test ?

Perhaps this happened when we JJB-ized it? https://github.com/theforeman/foreman-infra/blob/f21c2c920462d81ae9716382fb5f9b67e50fb340/puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/packaging_build_rpm.sh#L27 tells me we probably set the wrong properties.

**Comment by xprazak2** [comment]
I put the `0.3` back into the version

**Comment by ekohl** [comment]
I merged https://github.com/theforeman/foreman-infra/commit/7b8786aa90a273f3b5807283689b64fdcc55926c so after half an hour the slaves should have applied it and we can rerun the tests.

**Comment by ekohl** [comment]
[test packaging]

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -242,6 +242,7 @@
       <packagereq type="default">nodejs-select2</packagereq>
       <packagereq type="default">nodejs-style-loader</packagereq>
       <packagereq type="default">nodejs-symbol-observable</packagereq>
+      <packagereq type="default">nodejs-uglifyjs-webpack-plugin</packagereq>
```
Comment (ekohl) [inline comps/comps-foreman-rhel7.xml]: If you rebase on the current rpm/1.17 then this should already be present.

**Comment by ekohl** [comment]
Tests now pass. If you rebase this, I think this is good to go.

**Comment by xprazak2** [comment]
Rebased.

**Comment by mmoll** [comment]
merged, díky @xprazak2!

**Pull request merged**